UI/EncyclopediaDetailPanel.cpp

```diff
@@ -2680,8 +2680,10 @@ namespace {
             detailed_description += "\n";
         }
 
-        detailed_description += UserString("ENC_SUITABILITY_REPORT_WHEEL_INTRO")
-                                + "<img src=\"encyclopedia/EP_wheel.png\"></img>";
+        if (planet->Size() < SZ_ASTEROIDS) {
```

Comment:
The correct semantic would be to check against `Type`, not `Size`.

Reply:
fixed in a1a5683